var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -89,8 +89,6 @@ class Amrex(CMakePackage, CudaPackage, ROCmPackage):
     depends_on('hypre@2.19.0:', type='link', when='@21.03: ~cuda +hypre')
     depends_on('hypre@2.20.0:', type='link', when='@21.03: +cuda +hypre')
     depends_on('petsc', type='link', when='+petsc')
-    conflicts('%apple-clang')
-    conflicts('%clang')
```

Comment:
Is this a new feature? Previously I had build errors when building amrex with xcode/clang+brew/gfortran.

Or is this use case without fortran?

Reply:
Uh oh.  Too quick to approve.  Let me know if I need to revert.

Reply:
I see the prior change was at https://github.com/spack/spack/pull/8932

I don't remember the issues that came up at that time. [that resulted in that change]

The builds appear to work now [even with amrex@18.10.1 - where the prior issue came up] - so I guess this issue is nowwresolved [likely with updates on spack side]

Reply:
Hm, we test both Clang and AppleClang in CI.